pass.el

```diff
@@ -157,7 +163,8 @@ Similar to `save-excursion' but only restore the point."
 (defun pass-quit ()
   "Kill the buffer quitting the window."
   (interactive)
-  (when (y-or-n-p "Kill all pass entry buffers? ")
+  (when (or pass-suppress-confirmations
+             (y-or-n-p "Kill all pass entry buffers? "))
```

Comment:
Should we use `yes-or-no-p` here for consistency with the other prompts or use `y-or-n-p` in the other two?

Reply:
I think `y-or-n-p` should be used each time the operation is not dangerous (killing a saved buffer is not dangerous, deleting a file is).

Reply:
OK, then I think we can merge this change.